### PR TITLE
Simple cesium resource control

### PR DESCRIPTION
**Pull request by gberaudo**
This PR allows blocking Cesium rendering which is useful to save
resources. When no interaction is expected on the map, the application
can choose to block Cesium loop. Later on it may be unblocked.

### CODE REVIEW

**Comment by gberaudo** [comment]
Related issues in Cesium:
https://github.com/AnalyticalGraphicsInc/cesium/issues/2655 
https://github.com/AnalyticalGraphicsInc/cesium/issues/1865

**Comment by gberaudo** [comment]
Any review?

File: CHANGES.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
     primitive collection may be retrieved with the `getRootPrimitive()` method.
 
 * Changes
+  * Allow blocking Cesium rendering to save resources using
+    `olcs.OLCesium#blockCesiumRendering(true)`.
```
Comment (fredj) [inline CHANGES.md]: should be `olcs.OLCesium#setBlockCesiumRendering(true)`

**Comment by fredj** [comment]
+1, thanks

**Comment by gberaudo** [comment]
Thank you for the review @fredj.

**Pull request merged**